### ISSUE TITLE
Excel Add-in Inactive, not visible in Add-Ins toolbar

### ISSUE THREAD

**Issue by scrampy** [DaxStudio/DaxStudio#61]
Thanks for this great application - after using it for the past month, I can't imagine life without it!

I can run DaxStudio (2.7.4) standalone, but am unable to launch it from within Excel.  I'd like to be able to do so in order to enable the Output to Excel options.

Excel lists the add-in under Inactive Application Add-ins.  When I go to manage COM Add-ins, the add-in is visible, but un-checked.  If I check it and click OK, there is no effect.

I have uninstalled, then rebooted and reinstalled with no effect.

What steps should I try next?

DAX Studio Checker returns the following:

    Version =  v2.7.4.102

Checking Operating System
=======================
    OSCaption       = Microsoft Windows 10 Pro
    OSVersion       = 10.0.16299
    OSArchitecture  = 64-bit
    VisibleMemoryGB = 15.90
    FreeMemoryGB    = 7.17

    Culture Name              = en-CA
    Culture DisplayName       = English (Canada)
    Culture EnglishName       = English (Canada)
    Culture 2-Letter ISO Name = en
    Culture DecimalSeparator  = .
    Culture GroupSeparator    = ,
    Culture CurrencySymbol    = $
    Culture ShortDatePattern  = yyyy-MM-dd

Checking .Net Framework
=======================
    v2.0.50727 -> 2.0.50727.4927
    v2.0.50727\1033 -> 2.0.50727.4927
    v3.0 -> 3.0.30729.4926
    v3.0\Setup -> 3.0.30729.4926
    v3.0\Setup\1033 -> 3.0.30729.4926
    v3.0\Setup\Windows Communication Foundation -> 3.0.4506.4926
    v3.0\Setup\Windows Presentation Foundation -> 3.0.6920.4902
    v3.5 -> 3.5.30729.4926
    v3.5\1033 -> 3.5.30729.4926
    v4\Client -> 4.7.02556
    v4\Client\1033 -> 4.7.02556
    v4\Full -> 4.7.02556
    v4\Full\1033 -> 4.7.02556
    v4.0\Client -> 4.0.0.0

Checking AMO
=======================
    PASS > Microsoft.AnalysisServices, Version=13.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91
    PASS > Microsoft.AnalysisServices, Version=14.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91
    PASS > Microsoft.AnalysisServices, Version=15.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91

Checking ADOMD.NET
=======================
    PASS > Microsoft.AnalysisServices.AdomdClient, Version=13.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91
    PASS > Microsoft.AnalysisServices.AdomdClient, Version=14.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91
    WARN > Could not load file or assembly 'Microsoft.AnalysisServices.AdomdClient, Version=15.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

Dax Studio Configuration
========================
    Path: "C:\Program Files\DAX Studio\DaxStudio.exe"
    Processing DaxStudio.exe.config file...
    Path: 'C:\Program Files\DAX Studio\DaxStudio.exe.config'
    Bindings Found: 9
    AMO : 14.0.0.0
    ADOMD : 14.0.0.0

Checking Excel Add-in
=======================
    Detected Excel Version: 16 - Excel_v16
    Excel Path: C:\Program Files (x86)\Microsoft Office\Root\Office16\EXCEL.EXE
    Excel Architecture: x86

    DAX Studio Excel Add-in Registry keys
       Description: Dax Studio Excel Add-In
       FriendlyName: Dax Studio Excel Add-In
       Manifest: file:///C:/Program Files/DAX Studio/DaxStudio.vsto|vstolocal
       LoadBehavior: 3

    Checking for Excel 2010 Disabled Add-ins
      PASS > No Disabled items found.
    Checking for Excel 2013 Disabled Add-ins
      PASS > No Disabled items found.
    Checking for Excel 2016 Disabled Add-ins
      PASS > No Disabled items found.

    Attempting to load Excel Add-in: 
      PASS > Loaded Excel Add-in: DaxStudio, Version=2.7.4.102, Culture=neutral, PublicKeyToken=null

    Checking VSTO Configuration
    Architecture: x64
      v4
        Version 10.0.60301
        Install 1
        InstallerPath C:\Program Files (x86)\Common Files\Microsoft Shared\VSTO\10.0\VSTOInstaller.exe
      v4M
        VSTOMFeature_UNM 1
        VSTOMFeature_CLR35 1
        VSTOMFeature_CLR40 1

**Comment by scrampy** [comment]
I don't know what I did - but minutes after posting. I tried fighting with the add-ins dialog some more, and I now *can* see the add-in as expected!

The only thing that I can think of that I did differently was to close the empty new excel file that I was opening before going to File -> Options in Excel.  This time, I opened that menu with NO file open, set the add-in, then restarted and all is well.
